src/constants/domains.ts

```diff
@@ -0,0 +1,21 @@
+export const MIN_DOMAIN_NAME_LENGTH = 4;
+
+export enum BuyDomainStep {
+	Discover = 'Discover',
+	Approve = 'Approve',
+	Mint = 'Mint',
+	Success = 'Success',
+}
```

Comment:
Similar to a previous comment, you don't need the string values unless you're using the string itself (e.g. rendering the text).

Reply:
Updated